packages/wrangler/src/init.ts

```diff
@@ -56,6 +59,51 @@ interface InitArgs {
 	yes?: boolean;
 }
 
+export type ServiceMetaDataRes = {
```

Comment:
nit, but Metadata with a lowercase "d" is used in the backend—might be worth changing to that for consistency?

Reply:
Yeah that is actually one word and shouldn't be camelCase, I will change it.